test/install-content/custom_handler.sh

```diff
@@ -24,6 +24,14 @@ done
 exit_if_empty $RAUC_MOUNT_PREFIX
 mkdir -p $RAUC_MOUNT_PREFIX/image
 
+# This is only for testing
```

Comment:
Should this change be squashed into the one changing the code?

Reply:
Yes, could, was more ore less a post-fix.